packages/odyssey-design-tokens/src/color/functional.json

```diff
@@ -10,6 +10,15 @@
       "base": { "value": "{color.palette.red.500.value}" },
       "dark": { "value": "{color.palette.red.900.value}" }
     },
+    "caution": {
+      "base": { "value": "{color.palette.yellow.500.value}" }
+    },
+    "success": {
+      "base": { "value": "{color.palette.green.500.value}" }
+    },
+    "neutral": {
+      "base": { "value": "{color.palette.neutral.600.value}" }
+    },
```

Comment:
thanks for adding these. I'm hoping they are universal enough to live here

Reply:
These are pretty common. For other values, I figure we can use the rule of threes to decide when they become tokens / functional.

Reply:
What does the output/compiled name look like for these? My main question is whether or not `functional` (the file name) appears in the token name, it shouldn't, and I don't think it does -- but I don't remember.

Reply:
@arnoldsandoval-okta correct, it doesn't. the var here would be `var(--ColorNeutralBase)`